defaults/main.yml

```diff
@@ -14,6 +14,7 @@ jenkins_plugins: []
 jenkins_url_prefix: ""
 jenkins_java_options: "-Djenkins.install.runSetupWizard=false"
 jenkins_plugin_updates_expiration: 86400
+jenkins_plugins_timeout: 30
```

Comment:
Can you rename it to `jenkins_plugin_timeout` (singular instead of plural) here and elsewhere?

Also, one other thing that's missing is the addition of a documentation mention in README, with other variables, e.g.:

```
    jenkins_plugin_timeout: 30

The server connection timeout, in seconds, when installing Jenkins plugins.
```